app/src/components/basic-search/basic-search.js

```diff
@@ -84,7 +84,7 @@ const AddButton = ({ addFilter }) => {
         open={Boolean(anchorEl)}
         onClose={handleClose}
       >
-        {Object.keys(filterLabels).map(filter => (
+        {Object.keys(filterLabels).map((filter) => (
```

Comment:
Is there a reason all these arrow functions need parens around their parameters now?

Reply:
gonna look it into this

Reply:
in this process we upgraded major versions of prettier where they changed the default value for arrowParens from "avoid" to "always". updated .prettierrc.js to change it to the old default